It seems that this test fails when user.name is NOT set in ~/.gitconfig
on the build system.

--- a/test/config_test.rb
+++ b/test/config_test.rb
@@ -17,6 +17,7 @@
   end
 
   def test_read_global_config_file
+    skip 'This test is flawed, it runs on real FS, i.e. it is not isolated'
     config = Rugged::Config.global
     refute_nil config['user.name']
     assert_nil config['core.bare']
